tests/framework/PostProcessors/SparseSensing/testSPSLOptiTwistConstraints.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>tests/framework/Postprocessors/SparseSensing</name>
+    <author>Niharika Karnik (@niharika2999), Mohammad Abdo (@Jimmy-INL)</author>
+    <created>2024-04-01</created>
+    <classesTested>Models.Postprocessors.SparseSensing</classesTested>
+    <description>This test aims to check the ability of the Sparse Sensing Postprocessor
+     to locate and plot the selected sensors to reconstruct the temperature field of the OPTI-TWIST prototype
+     when perturbing the heater power in the presence of constraints. We assume the region near the heater to have very limited sensing capability and thus accomodate no sensors.
+    </description>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>reconstructionConstraintsOptiTwist</WorkingDir>
+    <Sequence>LoadCSV, mySPpp, print</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Files>
+    <Input name="twistFile" type="">./Perturbations2.csv</Input>
+    <Input name="refDO" type="">./350_330.csv</Input>
+  </Files>
+
+  <Steps>
+    <IOStep name="LoadCSV">
+      <Input class="Files" type="">twistFile</Input>
+      <Input class="Files" type="">refDO</Input>
+      <Output class="DataObjects" type="HistorySet">TwistDO</Output>
+      <Output class="DataObjects" type="PointSet">myDO</Output>
+    </IOStep>
+    <PostProcess name="mySPpp">
+      <Input class="DataObjects" type="HistorySet">TwistDO</Input>
+      <Model class="Models" type="PostProcessor">mySPSL</Model>
+      <Output class="DataObjects" type="DataSet">outPP</Output>
+    </PostProcess>
+    <IOStep name="print">
+      <Input class="DataObjects" type="DataSet">outPP</Input>
+      <Input class="DataObjects" type="PointSet">myDO</Input>
+      <Input class="DataObjects" type="HistorySet">TwistDO</Input>
+      <Output class="OutStreams" type="Print">outPP</Output>
+      <Output class="OutStreams" type="Plot">mySensorPlot</Output>
+      <Output class="OutStreams" type="Plot">mySensorPlot3D</Output>
+    </IOStep>
+  </Steps>
+
+  <Models>
+    <PostProcessor name="mySPSL" subType="SparseSensing" verbosity="debug">
+      <Goal subType="reconstruction">
+        <features>X (m),Y (m),Temperature (K)</features>  ### Try removing features, get X, Y from user
+        <measuredState>Temperature (K)</measuredState>
+        <basis>SVD</basis>
+        <nModes>4</nModes>
+        <nSensors>4</nSensors>
+        <nConstSensors>0</nConstSensors>
+        <constraintOption>"exact_n"</constraintOption>
+        <ConstrainedRegions type = "circle"> 
+          <center_x></center_x>  #center coordinates instead (x,y) (r, \theta)
+          <center_y></center_y>   
```

Comment:
Comments in xml are done using `<!--your comment-->`
center_x and center_y seems not to be recognized